### PR TITLE
deprecate ethercollateral loans

### PR DESCRIPTION
**Pull request by jacko125**
Minor audit feedback

### CODE REVIEW

**Review by justinjmoses — APPROVED** [review]
Just a note, even if we remove `EmptyEtherCollateral` it will still be referenced in `sources` for the contracts it was deployed for on L2...  Did we want to do this? I'm OK with it, just being explicit about this decision

File: contracts/BaseSynthetix.sol [hunk 1]
```diff
@@ -87,8 +87,6 @@ contract BaseSynthetix is IERC20, ExternStateToken, MixinResolver, ISynthetix {
         return issuer().totalIssuedSynths(currencyKey, false);
     }
 
-    // TODO: refactor the name of this function. It also incorporates the exclusion of
```
Comment (liamzebedee) [inline contracts/BaseSynthetix.sol:90]: The auditors told you to remove this comment? 😄 @jacko125

**Review by liamzebedee — APPROVED** [review]
This looks good.

**Comment by liamzebedee** [comment]
> Just a note, even if we remove `EmptyEtherCollateral` it will still be referenced in `sources` for the contracts it was deployed for on L2... Did we want to do this? I'm OK with it, just being explicit about this decision

What happens usually with these?

File: contracts/BaseSynthetix.sol (hunk 1 above)
Comment (jacko125) [reply contracts/BaseSynthetix.sol:90]: Yep the name of the function has been updated

**Pull request merged**